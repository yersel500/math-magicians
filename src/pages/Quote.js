import React from 'react';

const Quote = () => {
  return (
    <div>
      <p>Pure mathematics is, in its way, the poetry of logical ideas.
      </p>
    </div>
  );
}

export default Quote;