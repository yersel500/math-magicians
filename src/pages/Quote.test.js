import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from './Quote';

it('Quote page should render correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('Quote Page', () => {
  test('renders App component', () => {
    render(<Quote />);
    screen.debug();
  });
});
