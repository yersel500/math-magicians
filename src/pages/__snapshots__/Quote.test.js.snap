// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page should render correctly 1`] = `
<div
  className="quote-page"
>
  <p
    className="quote"
  >
    Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality. â€” Richard Courant, German-American mathematician
  </p>
</div>
`;
